scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,61 +2,31 @@
 
 # fail if some commands fails
 set -e
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
+fi
+if [[ -z $AMQP_URI ]]; then
+    echo "Please set AMQP_URI env with amqp uri or provide path of file containing it as ODO_RABBITMQ_AMQP_URL env"
+    exit 1
+fi
+export JOB_NAME="odo-pr-build"
+export REPO_URL="https://github.com/openshift/odo"
+export TARGET="$(jq .refs.pulls[0].number <<< $(echo $JOB_SPEC))"
+export CI_FIREWALL_VERSION="valpha"
+echo "Initiating"
 # show commands
 set -x
-
-ARCH=$(uname -m)
-export CI="openshift"
-if [ "${ARCH}" == "s390x" ]; then
-    make configure-installer-tests-cluster-s390x
-elif [ "${ARCH}" == "ppc64le" ]; then
-    make configure-installer-tests-cluster-ppc64le
-else
-    make configure-installer-tests-cluster
-fi
-make bin
-mkdir -p $GOPATH/bin
-make goget-ginkgo
-export PATH="$PATH:$(pwd):$GOPATH/bin"
-export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
-
-# Copy kubeconfig to temporary kubeconfig file
-# Read and Write permission to temporary kubeconfig file
-TMP_DIR=$(mktemp -d)
-cp $KUBECONFIG $TMP_DIR/kubeconfig
-chmod 640 $TMP_DIR/kubeconfig
-export KUBECONFIG=$TMP_DIR/kubeconfig
-
-# Login as developer
-odo login -u developer -p developer
-
-# Check login user name for debugging purpose
-oc whoami
-
-if [ "${ARCH}" == "s390x" ]; then
-    # Integration tests
-    make test-integration
-    make test-cmd-login-logout
-    make test-cmd-project
-    # E2e tests
-    make test-e2e-all
-elif  [ "${ARCH}" == "ppc64le" ]; then
-    # Integration tests
-    make test-integration
-    make test-cmd-login-logout
-    make test-cmd-project
-    # E2e tests
-    make test-e2e-all
+if [[ $CI_FIREWALL_VERSION == "master" ]]; then
+    git clone https://github.com/mohammedzee1000/ci-firewall.git cif
+    cd cif
+    git checkout main
+    go build -mod=vendor cmd/ci-firewall/ci-firewall.go
 else
-    # Integration tests
-    make test-integration
-    make test-integration-devfile
-    make test-cmd-login-logout
-    make test-cmd-project
-    make test-operator-hub
-    # E2e tests
-    make test-e2e-all
+    curl -kLO https://github.com/mohammedzee1000/ci-firewall/releases/download/${CI_FIREWALL_VERSION}/ci-firewall-linux-amd64.tar.gz
+    tar -xzf ci-firewall-linux-amd64.tar.gz
 fi
+./ci-firewall request --runscript scripts/run_all_tests.sh --timeout 45m
 
-odo logout
+exit 1
```

Comment:
This `exit 1` is to ensure logs are visible even with a successful run and needs to be removed before merging

Reply:
Can you please address this TODO?